.github/workflows/run-tests.yaml

```diff
@@ -0,0 +1,31 @@
+name: Tests
+
+on:
+  pull_request
```

Comment:
I _think_ this should work -- when I set the event trigger to "push" I get a clean build on Linux and macOS. I'm not sure how to test things when the trigger is set to `pull_request`. I feel like I'm overlooking something here...

Reply:
The GitHub Actions documentation isn't totally clear here; I think there may be an issue with creating a *new* workflow from a fork. I was expecting to see an option to approve running the workflow, but that's absent.